controllers/medusa/medusabackupjob_controller_test.go

```diff
@@ -405,3 +408,44 @@ func verifyObjectDoesNotExist(ctx context.Context, t *testing.T, f *framework.Fr
 		return err != nil && errors.IsNotFound(err)
 	}, timeout, interval, "failed to verify object does not exist", key)
 }
+
+func reconcileMedusaStandaloneDeployment(ctx context.Context, t *testing.T, f *framework.Framework, kc *k8ss.K8ssandraCluster, dcName string, k8sContext string) {
+	t.Log("check ReplicatedSecret reconciled")
+
+	medusaDepl := &appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      medusa.MedusaStandaloneDeploymentName(kc.SanitizedName(), dcName),
+			Namespace: kc.Namespace,
+		},
+		Spec: appsv1.DeploymentSpec{
+			Selector: &metav1.LabelSelector{
+				MatchLabels: map[string]string{"app": medusa.MedusaStandaloneDeploymentName(kc.SanitizedName(), dcName)},
```

Comment:
I'm a little confused about this one, shouldn't the kc.SanitizedName just be kc.meta.name?

Reply:
My understanding was that the SanitizedName was for getting a DNS-compatible k8s resource name from a Cassandra cluster name (which can contain more characters). But we should be naming dependant objects based on the k8s resource names, shouldn't we?

Reply:
As discussed offline, this is not the path we took with cluster/dc name overrides.
The SanitizedName will pick the override if it exists and sanitize it, or the resource name if not.
And all dependent objects will be named based on this.